.github/workflows/post-merge.yml

```diff
@@ -49,23 +51,29 @@ jobs:
             git add public/cards.json
             git commit -m "(CI) cards.json Generation"
           fi
-      - name: Format cards
-        if: env.SKIP_REST != 'true'
-        run: |
-          if [ ! -f .prettierrc ]; then
-            echo ".prettierrc not found"
-            echo "terminating workflow..."
-            exit 1;
-          fi
-          npx prettier public/cards.json --write
-          git add public/cards.json
-          git commit -m "(CI) Format cards" || echo "No changes to commit"
+
       - name: Generate screenshots
         if: env.SKIP_REST != 'true'
         run: |
           node generators/generateScreenshot.js
           git add Art/**/icon.png
           git commit -m "(CI) Screenshot Generation" || echo "No changes to commit"
+
       - name: Push changes
         if: env.SKIP_REST != 'true'
         run: git push origin master
+
+      - name: Run Prettier to fix code style
+        id: prettier-run
+        run: |
+          npm run prettier:fix
+          echo "filecount=$(git status -s -uno | wc -L)" >> $GITHUB_OUTPUT
+
+      - name: Commit Prettier changes
+        if: ${{ steps.prettier-run.outputs.filecount > 0 }}
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions - Prettier"
+          git add .
+          git commit -m "Run Prettier"
```

Comment:
nit: I think it might be nice to keep the commit message format more consistent with the others. i.e. maybe this could be "(CI) Run Prettier"?

Reply:
done